.github/workflows/ci.yaml

```
@@ -389,3 +389,29 @@ jobs:
       - run: ./devel/create-github-release "${{github.ref_name}}" dist/* nextstrain-cli-*-standalone-*
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  report-failure:
+    if: ${{ always() && github.ref_name == github.event.repository.default_branch }}
+    needs:
+      - lint
+      - test-source
+      - build-dist
+      - build-standalone
+      - test-dist
+      - test-standalone
+      - doc
+      - release
+    uses: nextstrain/.github/.github/workflows/report-failure.yaml
```

I believe the `@ref` is required by `uses:`.

```suggestion
    uses: nextstrain/.github/.github/workflows/report-failure.yaml@master
```